.ci/atime/tests.R

```diff
@@ -171,4 +171,25 @@ test.list <- atime::atime_test_list(
     Fast = "f248bbe6d1204dfc8def62328788eaadcc8e17a1"), # Merge commit of the PR (https://github.com/Rdatatable/data.table/pull/6296) that fixes the issue
 
   NULL)
+
+extra.args.6107 <- c(
+  "colClasses=list(Date='date')",
+  "colClasses='Date'",
+  "select=list(Date='date')")
+for(extra.arg in extra.args.6107){
```

Comment:
@MichaelChirico what do you think about this refactor using a for loop?
I think it clearly emphasizes the similarities and differences between the three proposed test cases.
is it ok with you?

Reply:
Yes, very nice idea!